src/notation/internal/notationparts.cpp

```diff
@@ -37,15 +37,39 @@ using namespace mu::engraving;
 
 static const mu::engraving::Fraction DEFAULT_TICK = mu::engraving::Fraction(0, 1);
 
+static QString formatInstrumentTitleOnScore(const QString& instrumentName, const Trait& trait)
+{
+    // Comments for translators start with //:
```

Comment:
I think it is sensible to _not_ say what `//:` does except in a few key places where it is useful and doesn't lead to unnecessary bloat. I think this is one of those places where it is worth saying because these format strings are vitally important and the comments here really are essential.